Project/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs

```diff
@@ -18,11 +19,27 @@ public enum Team
         Purple = 1
     }
 
+    public enum Position
+    {
+        Striker,
+        Goalie,
+        Generic
+    }
+
     [HideInInspector]
     public Team team;
     float m_KickPower;
     int m_PlayerIndex;
     public SoccerFieldArea area;
+    float m_BallTouch;
+    public Position position;
+
+    float m_Power;
```

Comment:
```suggestion
    const float k_Power = 2000f;
```
I think this is better as a const.